src/osdc/Objecter.h

```diff
@@ -446,9 +446,10 @@ struct ObjectOperation {
 	    for (list<watch_item_t>::iterator i = resp.entries.begin() ;
 		 i != resp.entries.end() ; ++i) {
 	      obj_watch_t ow;
-	      ostringstream sa;
-	      sa << i->addr;
-	      strncpy(ow.addr, sa.str().c_str(), 256);
+              ostringstream ss;
+              ss << i->addr;
+	      strncpy(ow.addr, ss.str().c_str(), 256);
+              ow.addr[255] = '\0'; // make coverity happy
```

Comment:
`strncpy()` does include the terminating `\0`. and `ss.str().c_str()` evaluates a null-terminated c string.